File: API/WebApplication1/Models/Rolle.cs
namespace WebApplication1.Models
{
    public class Rolle : Common
    {
        public required string Navn { get; set; }
        public List<Egenskab?> Egenskaber { get; set; } = new List<Egenskab?>();
    }
    public class RolleDTO
    {
        public required string navn { get; set; }
        public List<string> EgenskabIds { get; set; } = new List<string>();
    }
}

File: Flutter/assignly/lib/classes/objects/rolle.dart
import 'package:assignly/classes/objects/egenskab.dart';
import 'dart:convert';

class Rolle {
  String? id, navn;
  List<Egenskab?> egenskaber = [];

  static Rolle getRolleFromJson(String response) {
    try {
      final data = jsonDecode(response);
      final results = data;

    } catch (_) {
    }
    return Rolle();
  }
}